docs/_plugins/sidebar_item.rb

```diff
@@ -4,7 +4,14 @@ def sidebar_item_link(item)
       pageID = @context.registers[:page]["id"]
       itemID = item["id"]
       href = item["href"] || "/react/docs/#{itemID}.html"
-      className = pageID == itemID ? ' class="active"' : ''
+      classes = []
+      if pageID == itemID
+        classes.push("active")
+      end
+      if item["href"]
+        classes.push("external")
+      end
+      className = classes.size ? "class=\"#{classes.join(' ')}\"" : "";
```

Comment:
This isn't right, here's what gets generated for empty items. I have another thing I want to change so going to follow up.

```diff
           <li>
-            <a href="/react/docs/webcomponents.html">Web Components</a>
+            <a href="/react/docs/webcomponents.html"class="">Web Components</a>

           </li>
```